sdk/storage/azure-storage-queue/src/main/java/com/azure/storage/queue/QueueAsyncClient.java

```diff
@@ -655,14 +657,14 @@ public String getQueueUrl() {
      *
      * <p>Peek the first message</p>
      *
-     * {@codesnippet com.azure.storage.queue.queueAsyncClient.peekMessages}
+     * {@codesnippet com.azure.storage.queue.queueAsyncClient.peekMessage}
      *
      * <p>For more information, see the
      * <a href="https://docs.microsoft.com/en-us/rest/api/storageservices/peek-messages">Azure Docs</a>.</p>
      *
-     * @return A {@link PeekedMessage} that contains metadata about the message.
+     * @return A {@link PeekedMessageItem} that contains metadata about the message.
      */
-    public PagedFlux<PeekedMessage> peekMessages() {
+    public PagedFlux<PeekedMessageItem> peekMessage() {
```

Comment:
Same here. Should this be a `Mono` instead of a `PagedFlux` as it's returning a single message?